components/style/properties/properties.mako.rs

```diff
@@ -435,7 +435,8 @@ impl Shorthand {
     pub fn serialize_shorthand_to_buffer<'a, W, I>(self,
                                                    dest: &mut W,
                                                    declarations: I,
-                                                   is_first_serialization: &mut bool)
+                                                   is_first_serialization: &mut bool,
+                                                   is_important: bool)
```

Comment:
Can we pass an `Importance` argument instead? Not a huge deal, but I think it's a bit clearer. Don't bother too much about it if it's too much churn.